frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelinesTable.ts

```diff
@@ -33,8 +49,16 @@ class PipelinesTable {
     );
   }
 
-  toggleExpandRowByIndex(index: number) {
-    cy.findByLabelText('Details').eq(index).click();
+  toggleExpandRowByName(name: string) {
+    this.findRowByName(name).findByLabelText('Details').click();
+  }
+
+  toggleCheckboxByRowName(name: string) {
+    this.findRowByName(name).findByRole('checkbox').click();
+  }
+
+  shouldBeEmpty() {
+    return cy.findByTestId('global-no-pipelines');
```

Comment:
This isn't actually checking if the table should be empty. I believe updating this to be return `cy.findByTestId('global-no-pipelines').should('not.exist')`

should fix your build issue, and actually makes it so that the name of the function reflects the operations within that function.